docs/user-guide/deployments-administration/deploy-on-kubernetes/overview.md

```diff
@@ -5,25 +5,39 @@ description: Overview of deploying GreptimeDB on Kubernetes using the GreptimeDB
+## Deploy GreptimeDB Standalone
 
-We **highly recommend** using the GreptimeDB Operator to deploy GreptimeDB on Kubernetes.
+For development, testing, or small-scale production use cases, you can [deploy a standalone GreptimeDB instance](deploy-greptimedb-standalone.md) on Kubernetes.
+This provides a simple way to get started with GreptimeDB without the complexity of managing a full cluster.
 
-:::warning
-If you are not using GreptimeDB Operator to deploy/upgrades the cluster, please refer to [Cluster Maintenance Mode](/user-guide/deployments-administration/maintenance-mode.md) for required operations during deployment and cluster upgrades.
-:::
+## Deploy GreptimeDB Cluster
 
-## Getting Started
+For production environments requiring high availability and scalability,
+you can [deploy a GreptimeDB cluster](deploy-greptimedb-cluster.md) using the GreptimeDB Operator on Kubernetes.
+This enables you to set up a distributed GreptimeDB cluster that scales horizontally and efficiently handles large volumes of data.
 
-- GreptimeDB Cluster: You can take [Getting Started](./deploy-greptimedb-cluster.md) as your first guide to understand the whole picture. This guide provides the complete process of deploying the GreptimeDB cluster on Kubernetes.
-- [GreptimeDB Standalone](./deploy-greptimedb-standalone.md)
+## Configurations
 
-## GreptimeDB Operator
+You can apply custom configurations to GreptimeDB by creating a `values.yaml` file
+when deploying either GreptimeDB clusters or standalone instances.
+For a complete list of available configuration options, see [Common Helm Chart Configurations](./common-helm-chart-configurations.md).
 
-- [GreptimeDB Operator Management](./greptimedb-operator-management.md)
+## Manage GreptimeDB Operator
 
-## Configurations
+The GreptimeDB Operator manages GreptimeDB deployments on Kubernetes,
+automating the setup, provisioning, and management of GreptimeDB cluster instances.
+This enables quick deployment and scaling of GreptimeDB in any Kubernetes environment,
+whether on-premises or in the cloud.
+Learn how to [manage the GreptimeDB Operator](./greptimedb-operator-management.md),
+including installation and upgrades.
+
+## Advanced Deployments
+
+After familiarizing yourself with [the architecture and components of GreptimeDB](/user-guide/deployments-administration/architecture.md), you can explore advanced deployment scenarios:
+
+- [Deploy GreptimeDB Cluster with Remote WAL](configure-remote-wal.md): Configure Kafka as a remote write-ahead log (WAL) for your GreptimeDB cluster to persistently record every data modification and ensure no loss of memory-cached data.
```

Comment:
In kubernetes, use: https://docs.greptime.com/nightly/user-guide/deployments-administration/deploy-on-kubernetes/common-helm-chart-configurations#enable-remote-wal

Reply:
The content in this document can be applicable to the k8s environment:
https://docs.greptime.com/nightly/user-guide/deployments-administration/deploy-on-kubernetes/configure-remote-wal

Reply:
Vote to `configure-remote-wal.md`